.travis.yml

```diff
@@ -26,6 +27,14 @@ jobs:
       services: postgresql
       script: jdk_switcher use oraclejdk8 && ext/travisci/run-core-tests
 
+    # OracleJDK9
```

Comment:
Don't feel strongly about it, but I'd be tempted to either leave this header comment and the JDK8 one off, under the argument that the PDB_TEST indicators are prominent enough, or to generalize them to "JDK8" and "JDK9" "group" headers.

Reply:
Ah I didn't mean to leave those in, just had those in there while I was making sure I ran all the relevant tests with jdk9